libsmtutil/CHCSmtLib2Interface.cpp

```diff
@@ -96,7 +101,10 @@ std::tuple<CheckResult, Expression, CHCSolverInterface::CexGraph> CHCSmtLib2Inte
 	CheckResult result;
 	// TODO proper parsing
 	if (boost::starts_with(response, "sat"))
-		result = CheckResult::UNSATISFIABLE;
+	{
```

Comment:
Is the comment in line 102 still relevant?

Reply:
Not sure what it means, actually. It has been there since the origin of `CHCSmtLib2Interface`.
I think with this PR we can parse models (invariants) from CHC solvers, but I still need to implement parsing proofs (counterexamples). Maybe then we can remove this TODO.